jfr-mappers/src/main/java/com/newrelic/jfr/toevent/ThreadLockEventMapper.java

```diff
@@ -9,7 +9,7 @@
 
 import static com.newrelic.jfr.RecordedObjectValidators.hasField;
 
-import com.newrelic.jfr.MethodSupport;
+import com.newrelic.jfr.profiler.MethodSupport;
```

Comment:
Given that classes like this also need to use MethodSupport, why did we move it into the profiler package?

Reply:
🦅  👁️  moved